doc/nuklear.html

```diff
@@ -2313,6 +2314,17 @@
    - [x]: Major version with API and library breaking changes
    - [yy]: Minor version with non-breaking API and library changes
    - [zz]: Bug fix version with no direct changes to API
+- 2020/07/06 (4.04.0) - Added subpixel API from itsuhane. Allowing you to turn on 
+                        NK_ENABLE_SUBPIXEL_API to position window and elements in subpixels
+                        Changed nk_begin to return a nk_window so you can get the position and other useful info.
+                        Fixed radio buttons' style.
+                        Fixed click cascading through multiple buttons; it should only click the upper button.
+                        Strict c++17 now compiles: only declaring memset, memcpy if they are used.
+                        Only asserting index range if ushorts are used for indices.
+                        Made paq.sh identical to paq.bat (outputs to nuklear.h).
+                        Replaced NK_ASSERT(0 && ...) with NK_ERROR(), to ensure it compiles on strict C++17.
+                        Replaced const char** by const char * const * in combobox, to ensure const List&lt;const char*&gt; can be used.
+                        Added nk_tree_type for allowing the symbol next to a tree element to be hidden (for children) if it's not an image.
 - 2020/04/09 (4.02.1) - Removed unused nk_sqrt function to fix compiler warnings
                       - Fixed compiler warnings if you bring your own methods for
```

Comment:
Hmm. The versioning in here seems quite out of date

Reply:
I think we'll fix the versioning after we finally know which of the (many) other PRs get merged first.